openapidoc.go

```diff
@@ -0,0 +1,15 @@
+package main_test
```

Comment:
Rename to `package commerce` (as discussed)

Reply:
Done - pleas recheck